nagiosplugin/check.py

```diff
@@ -27,19 +27,24 @@
 
 class Check(object):
 
-    def __init__(self, *objects):
+    def __init__(self, *objects, **kwargs):
         """Creates and configures a check.
 
         Specialized *objects* representing resources, contexts,
         summary, or results are passed to the the :meth:`add` method.
         Alternatively, objects can be added later manually.
+        If no *name* is given, the output prefix is set to the first
+        resource's name. If *name* is None, so prefix is set at all.
```

Comment:
Is this a typo?  ITYM:
```suggestion
        resource's name. If *name* is None, no prefix is set at all.
```

Reply:
(Yes, this is a typo.)

Reply:
OK.  Do you want to push a change to your PR to fix it?  Once that's done I'll merge it in.

Reply:
I think it's already fixed in 98e786a4, isn't it?